code/modules/mob/living/living.dm

```diff
@@ -1557,6 +1558,10 @@ Thanks.
 		to_chat(src, "<span class='warning'>You can't do that now!</span>")
 		return FAILED_THROW
 
+	if(runescape_pvp && is_pacified())
+		to_chat(src, "<span class='warning'>As such, throwing items is also forbidden outside of maint.</span>")
```

Comment:
Informal "maint" shouldn't appear in system messages

also, "as such"?

Reply:
"As such" because is_pacified() already sends a message when trying to attack outside of maint when this mode is enabled.

Reply:
```suggestion
		to_chat(src, "<span class='warning'>As such, throwing items is also forbidden outside of maintenance areas.</span>")
```